resources/js/pages/NewAddress.vue

```diff
@@ -0,0 +1,66 @@
+<template>
+  <div class="home">
+
+    <form @submit.prevent= "handleSubmit" >
+
+      <inputlabel label_text="Building" v-model="building"/>
+      <inputlabel label_text="Apartment" v-model="apartment"/>
+      <inputlabel label_text="Floor" v-model="floor"/>
+      <inputlabel label_text="Street" v-model="street"/>
+      <inputlabel label_text="Email" v-model="email"/>
```

Comment:
add validation on the email input, to only accept correct email string other wise show an error msg